operator/v1/types_console.go

```diff
@@ -40,6 +40,7 @@ type ConsoleSpec struct {
 	// The default console route will be maintained to reserve the default hostname
 	// for console if the custom route is removed.
 	// If not specified, default route will be used.
+	// DEPRECATED: will be removed in 4.10
```

Comment:
I'd just say "DEPRECATED" for now. I don't think we've decided that we will remove it in a specific release.

@ahardin-rh We should document this as deprecated in the OpenShift docs as well. This relates to https://issues.redhat.com/browse/CONSOLE-2496. Thanks!

Reply:
PR already opened :+1 
https://github.com/openshift/openshift-docs/pull/32410